UnoCheck/Solutions/HyperVActivationSolution.cs

```diff
@@ -0,0 +1,45 @@
+﻿using DotNetCheck.Models;
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace DotNetCheck.Solutions
+{
+	public class HyperVActivationSolution : Solution
+	{
+
+		public HyperVActivationSolution()
+		{
+
+
+		}
+
+		public override async Task Implement(SharedState sharedState, CancellationToken cancellationToken)
+		{
+			await base.Implement(sharedState, cancellationToken);
+
+			if (await this.ActivateHyperVAsync())
+			{
+				ReportStatus("Activating Hyper-V. Please, restart computer to complete HyperV activation process.");
+			}
+		}
+
+		private async Task<bool> ActivateHyperVAsync()
+		{
+			var startInfo = new ProcessStartInfo
+			{
+				Arguments = "/set hypervisorlaunchtype auto",
+				CreateNoWindow = true,
+				FileName = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.System), "bcdedit.exe"),
+				UseShellExecute = false
```

Comment:
I think we should only point to Microsoft's documentation, there are multiple machine specific steps that may need to be taken. The other validations are useful though, thank you.